SPECS/ocaml-gnt.spec

```diff
@@ -30,6 +31,7 @@ or network backends.
 %package        devel
 Summary:        Development files for %{name}
 Requires:       %{name} = %{version}-%{release}
+Requires:       ocaml-mirage-profile-devel
```

Comment:
All of these Requires: lines for the devel package should have a %{?_isa} suffix.

https://fedoraproject.org/wiki/Packaging:Guidelines#Package_dependencies